src/Extensions/FormAnswersExtensions.cs

```diff
@@ -12,7 +14,7 @@ public static List<PageAnswers> GetReducedAnswers(this FormAnswers answer, FormS
                 answer.Pages,
                 answer.Pages.SelectMany(_ => _.Answers).ToDictionary(x => x.QuestionId, x => x.Response),
                 schema.Pages,
-                schema.FirstPageSlug,
+                String.IsNullOrEmpty(answer.Path) ? schema.FirstPageSlug : answer.Path.Contains(FileUploadConstants.DOCUMENT_UPLOAD_URL_PATH) ? $"{answer.Path}" : schema.FirstPageSlug,
                 new List<PageAnswers>());
```

Comment:
contains would give a false positive for not-document-upload or document-upload-infomation, this should just be Equals? also could this not be a single ternary with && if u need to check if its notnull first.